kalite/distributed/management/commands/restorebackup.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+from django.core.management.base import BaseCommand
+from django.core.management import call_command
+from django.conf import settings
+
+from optparse import make_option
+
+class Command(BaseCommand):
+    help = "Restore backup to a previous state."
+    option_list = BaseCommand.option_list + (
+        make_option(
+            '--file',
+            dest='backup_filepath',
+            default="",
+            help="File path"),
+        )
+    
+    def handle(self, *args, **options):
+
+            if(options['backup_filepath']):
+                backup_filepath = options['backup_filepath']
+                if(os.path.isfile(backup_filepath)):
+                    call_command('dbrestore', filepath=backup_filepath, database='default') #perform restore from the given filepath
+                else:
+                    print 'File not found'
+            else:
+                file_list = os.listdir(settings.BACKUP_DIRPATH) #Retrieve the filenames present in the BACKUP_DIRPATH
+                filelist = []
+                filenumber = 0
+                for i, f in enumerate(file_list):
+                    print i, f.replace(".backup", "")
+                    filelist.append(f)
+                if(len(filelist) > 0):        
```

Comment:
First off:

Unlike the other *uncool* languages, python does not need parens in its if statements, unless you really need it to layout the order of evaluation. So this statement becomes:

`if len(filelist) > 0:`

Python also treats any value that's not an empty list, empty dict, empty set, 0 or `False` as a `True` value. So then your if statement can be restated to:

`if len(filelist):`

However, since a nonempty list is treated as `True`, you can reduce this to:

`if filelist`:

Much simpler and pythonic! Can you go over your if statements in this PR and see if you can simplify them?